LICENSE.md

```diff
@@ -1,5 +1,6 @@
 The MIT License (MIT)
 
+Copyright (c) 2017 stakx
 Copyright (c) 2013 Imazen
```

Comment:
All other original copyright notices mention the year 2012, not sure why it says 2013 here. However, I don't feel to comfortable changing a preexisting copyright notice, so I'll leave this as it was for now.